test/Dialect/FIRRTL/SFCTests/gct-signal-mappings/sub_circuits/other/circuit.fir

```diff
@@ -0,0 +1,49 @@
+; RUN: true
```

Comment:
What's the plan with this? 😄

Reply:
:grin: That's to prevent `llvm-lit` from complaining about the file not having a `RUN` line. I thought you could just mark files to be ignored, but I haven't been able to find that in the documentation yet.

Reply:
This is now cleaned up.